script/client/conf/file.conf

```diff
@@ -68,4 +69,27 @@ client {
   log {
     exceptionRate = 100
   }
-}
\ No newline at end of file
+}
+
+saga {
+  enabled = false
+  stateMachine {
+    applicationId = ""
```

Comment:
Why configure multiple times?

Reply:
After another PR adds the relevant code, I will delete it here.

Reply:
I have deleted it. The config is read in the PR #2831